src/py21cmfast/src/SpinTemperatureBox.c

```diff
@@ -1434,7 +1766,17 @@ LOG_SUPER_DEBUG("finished loop");
             Xheat_ave /= (double)HII_TOT_NUM_PIXELS;
             Xion_ave /= (double)HII_TOT_NUM_PIXELS;
 
-            LOG_DEBUG("zp = %e Ts_ave = %e x_e_ave = %e Tk_ave = %e J_alpha_ave = %e xalpha_ave = %e Xheat_ave = %e Xion_ave = %e",zp,Ts_ave,x_e_ave,Tk_ave,J_alpha_ave,xalpha_ave,Xheat_ave,Xion_ave);
+            if (flag_options->USE_MINI_HALOS){
+                J_alpha_ave_MINI /= (double)HII_TOT_NUM_PIXELS;
+                Xheat_ave_MINI /= (double)HII_TOT_NUM_PIXELS;
+                J_LW_ave /= (double)HII_TOT_NUM_PIXELS;
+                J_LW_ave_MINI /= (double)HII_TOT_NUM_PIXELS;
+
+                LOG_DEBUG("zp = %e Ts_ave = %e x_e_ave = %e Tk_ave = %e J_alpha_ave = %e(%e) xalpha_ave = %e Xheat_ave = %e(%e) Xion_ave = %e J_LW_ave = %e (%e)",zp,Ts_ave,x_e_ave,Tk_ave,J_alpha_ave,J_alpha_ave_MINI,xalpha_ave,Xheat_ave,Xheat_ave_MINI,Xion_ave,J_LW_ave/1e21,J_LW_ave_MINI/1e21);
```

Comment:
Its probably best to have the mini halo properties on their own line. It'll be easier to read then.